### DISCUSSION TITLE
Notice to GoDaddy Users: GoDaddy DNS API will no longer work for customers will less than 10 domains.

### DISCUSSION THREAD

**Discussion by webprofusion-chrisc** [acmesh-official/acme.sh#5182] [General]
As per the following issues, GoDaddy have changed their API and it will reject operations for users with less than 10 domains managed on GoDaddy. If you have a problem with GoDaddy speak to their support. **This is not a bug in acme.sh** or any other ACME client.

https://github.com/acmesh-official/acme.sh/issues/5181
https://github.com/acmesh-official/acme.sh/issues/4487
https://github.com/acmesh-official/acme.sh/issues/5178
Etc.

From GoDaddy Support:
![340103030-a1bf2ac3-ccd8-4bdd-9294-95536c927d30](https://github.com/acmesh-official/acme.sh/assets/2445502/14e78645-d88c-4119-92fc-6bafa0bf7174)

**Comment by rossta0825** [comment]
Interesting to note as a GoDaddy user who spent days trying to debug why I could not get a new certificate to issue with acme.sh - although the GoDaddy API was giving an error of Not Authorized, it was actually allowing the TXT records to be added.  This really threw me off because I could see the required TXT record being added and yet the script was failing.  The script was also not respecting the --dnssleep parameter so I was focused on how to get it to wait longer for the TXT record to propagate. But, I think acme.sh had already decided it had failed even though it continued to issue commands and report through the --debug 2 option.  

I have 2 different accounts with 6 domains in each that GoDaddy will be seeing go away due to this.

**Reply by alanialgor** [reply]
OMG, This is exactly what I am going through.  Been Banging my head for weeks.  Off and on that is.  hehe
I have 1 domain with GoDaddy.  It is mainly so I can do Resolver in my intranet with domains.
I created second level subdomains in my Resolver and was adding those to the cert before.  Then it stopped working.  Thought it was because of that and the fact I was doing wild cards.
Like yourself I could see the text being added and the authorization error.  was confused as heck.
I called tech support, got a fella on the line and was about to ask about the acme challenge when all I could hear in the background was a steady banging.  Like at a house being built.  I was unable to converse, I asked if he could tell the maintenance guy to stop the banging for a few minutes, he says, "it's my dad, he is doing something."  I'm like oh you are WFH.  

Then while the banging was continuing I was trying to get info from him and was getting nowhere.  As I asked to speak to someone else who may know something about this, the dog chimed in.  Dog barking, dad hammering on the walls, continuously ( loud too ), no info, complete waste of time.
call ended

What is another option?  Is there a DNS provider that I could use that would make this work again?

P.S.  If I look in the tmp directory where the script is preparing files while negotiating with GoDaddy DNS I see there is a csr file and it ends up being created with the new info.  But because the script fails on a verify it doesn't seem to finish and put the necessary files in the acme folder.  maybe I can use these?  do the conversions, for my needs?
